src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,208 @@
+                    + car.getModel() + ", manufacturer_id - "
+                    + car.getManufacturerId(), e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        Car car = null;
+        String query = "SELECT cars.id as carId, cars.model, "
+                + "manufacturers.id as manufacturerId, manufacturers.name as manufacturer, "
+                + "manufacturers.country "
+                + "FROM cars "
+                + "INNER JOIN manufacturers "
+                + "ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars.id = ? AND cars.is_deleted = false";
+        Connection connection = ConnectionUtil.getConnection();
+        try (PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                car = collectCar(resultSet);
+            }
+            return Optional.ofNullable(car);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get car with id - " + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String query = "SELECT cars.id as carId, cars.model, "
+                + "manufacturers.id as manufacturerId, manufacturers.name as manufacturer, "
+                + "manufacturers.country "
+                + "FROM cars "
+                + "INNER JOIN manufacturers "
+                + "ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars.is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            ResultSet resultSet = statement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                Car car = collectCar(resultSet);
+                car.setDrivers(getDriversFromCar(car));
+                cars.add(car);
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get list of cars", e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        List<Car> cars = new ArrayList<>();
+        String query = "SELECT drivers.id, drivers.name, drivers.license_number, "
+                + "manufacturers.id as manufacturerId, "
+                + "manufacturers.name as manufacturer, manufacturers.country, "
+                + "cars.id AS carId, cars.model "
+                + "FROM  cars_drivers "
+                + "INNER JOIN cars on cars_drivers.car_id = cars.id "
+                + "INNER JOIN manufacturers on cars.manufacturer_id = manufacturers.id "
+                + "INNER JOIN drivers on cars_drivers.driver_id = drivers.id "
+                + "WHERE driver_id = ?";

```

Comment:
You can simplify it, don't use inner join, for this query it's enough 2 simple joins. Once again use aliases